packages/adapters-library/src/core/provider/MulticallQueue.ts

```diff
@@ -59,6 +59,9 @@ export class MulticallQueue {
     this.provider = new ethers.JsonRpcProvider(
       fetchRequest,
       Network.from(chainId),
+      {
+        staticNetwork: Network.from(chainId),
+      },
```

Comment:
This was causing us to spam infura on every `eth_call` with an `eth_chainId`.

Reply:
nice save